cliquet/listeners/redis.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+from six.moves.urllib import parse as urlparse
+import redis
+import json
+
+from cliquet.listeners import ListenerBase
+
+
+class RedisListener(ListenerBase):
+    def __init__(self, *args, **kwargs):
+        super(RedisListener, self).__init__(*args, **kwargs)
+        maxconn = kwargs.pop('max_connections')
+        self._listname = kwargs.pop('listname', 'cliquet.events')
+        connection_pool = redis.BlockingConnectionPool(max_connections=maxconn)
+        self._client = redis.StrictRedis(connection_pool=connection_pool,
+                                         **kwargs)
+
+    def __call__(self, event):
+        try:
+            payload = json.dumps(event.payload)
+        except Exception:
```

Comment:
Yes but since the listeners are executed synchronously it would be equivalent and would apply to every listeners. I guess I prefer your solution, since we may to let listeners trigger some httpexceptions for example